subscribed_topics:
  rgb_image_topic: synchronized/rgb/image

published_topics:
  candidate_holes_topic: synchronized/rgb/candidate_holes
